src/core/reactor.cc

```diff
@@ -2369,15 +2369,22 @@ void reactor::run_tasks(task_queue& tq) {
     }
 }
 
+namespace {
+
 #ifdef SEASTAR_SHUFFLE_TASK_QUEUE
-void reactor::shuffle(task*& t, task_queue& q) {
+void shuffle(task*& t, circular_buffer<task*>& q) {
     static thread_local std::mt19937 gen = std::mt19937(std::default_random_engine()());
-    std::uniform_int_distribution<size_t> tasks_dist{0, q._q.size() - 1};
-    auto& to_swap = q._q[tasks_dist(gen)];
+    std::uniform_int_distribution<size_t> tasks_dist{0, q.size() - 1};
+    auto& to_swap = q[tasks_dist(gen)];
     std::swap(to_swap, t);
 }
+#else
+void shuffle(task*& t, circular_buffer<task*>& q) {
+}
```

Comment:
```suggestion
void shuffle(task*&, circular_buffer<task*>&) {
}
```

otherwise a picky compiler would complain that variables defined by not used.

Reply:
Will do